data/data/manifests/bootkube/openshift-service-ca-signer-secret.yaml.template

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: service-ca-signing-key
+  namespace: openshift-service-ca-operator
+type: kubernetes.io/tls
+data:
+  tls.crt: {{.ServiceServingCaCert}}
```

Comment:
nit: why is this `Ca` not `CA`? the inconsistency is bothering me! Can we change it to `{{.ServingCACert}}` while we're at it too?

Reply:
I'd prefer to leave this template variable name as-is for this PR. We get rid of the SSCS name, then ServingCACert can be its final resting name.